src/htmlize/patchtml.ml

```diff
@@ -20,10 +19,139 @@ let edit_html ?header ?footer ~head_childs html =
     footer
     "#footer";
   let head = html $ "head" in
-  List.iter (fun (id,child) -> 
-      opt_inserter
-        (iter (fun e -> append_child head e))
-        (Some child)
-        id)
+  List.iter (fun (id,child) ->
+    opt_inserter
+      (iter (fun e -> append_child head e))
+      (Some child)
+      id)
     head_childs;
   to_string html
+
+open Omd
+(* Keeps us from processing url links *)
+let is_url url =
+  let http = String.sub url 0 7 in
+  if String.equal http "http://" then
+    true
```

Comment:
`if x then true` is usually a bad thing. ;)

This could be written:

```
let is_url url =
  String.equal (String.sub url 0 7) "http://" || String.equal (String.sub url 0 8)
```

Reply:
Yeah my bad haha